iojs.org/aliases.json

```diff
@@ -115,5 +115,18 @@
       "gibfahn@gmail.com",
       "hello@bnb.im",
       "rtrott@gmail.com"
+  ] },
+    { "from": "cve-request", "to": [
+      "michael_dawson@ca.ibm.com",
+      "info@bnoordhuis.nl",
+      "viewtech@gmail.com"
```

Comment:
`viewtech@gmail.com`? Huh? What?

Reply:
That is Sam roberts

Reply:
I believe his email is vieuxtech@gmail.com rather than viewtech@gmail. At least, that first email is what appears in our git logs.

Reply:
:-)  Yes, good catch, @mhdawson - pls fix the spelling and land!

Reply:
(Note that it appears in two places so correct in both, of course.)